src/libraries/System.Private.CoreLib/src/System/Text/Ascii.cs

```diff
@@ -189,31 +186,33 @@ public static partial class Ascii
                         return false;
                     }
 
+                    nuint i = 4 * (nuint)Vector128<T>.Count;
+
                     searchSpace = ref Unsafe.Add(ref searchSpace, 4 * Vector128<T>.Count);
 
                     // Try to opportunistically align the reads below. The input isn't pinned, so the GC
                     // is free to move the references. We're therefore assuming that reads may still be unaligned.
                     // They may also be unaligned if the input chars aren't 2-byte aligned.
                     nuint misalignedElements = ((nuint)Unsafe.AsPointer(ref searchSpace) & (nuint)(Vector128<byte>.Count - 1)) / (nuint)sizeof(T);
-                    searchSpace = ref Unsafe.Subtract(ref searchSpace, misalignedElements);
+                    i -= misalignedElements;
```

Comment:
As `i` is unsigned native integer, and being a bit paranoid, I'd add something like
```c#
Debug.Assert((int)i >= 0);
```
here. So to assert hat no underflow happened.

Reply:
It'll be `32 - ((someUnsignedValue & 15) / 2)` for chars, so I would hope so 😄
Added an assert that `i > 3 * Vector128<T>.Count` in this case.